inst/sympref.m

```diff
@@ -428,7 +431,8 @@
 %!test
 %! syms x
 %! r = sympref('reset');
+%! % restore original sympref settings
+%! sympref ('ipc',   sympref_orig.ipc);
+%! sympref ('quiet', sympref_orig.quiet);
 %! syms x
 %! assert(r)
-%! % ok, can be noisy again
-%! sympref('quiet', 'default')
```

Comment:
I think its ok.  Maybe line 434 is not needed as resetting the IPC does that anyway, but its not a big deal.